allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
ok... missed this case. So we need to add line-height to concrete blocks? But think it can be done in separate pr